### PR TITLE
Add generated labels to MathJax in Storybook

### PR DESCRIPTION
**Pull request by nedredmond**
Adds generated labels to MathJax in Storybook so that we can effectively test accessibility without involving webapp.

Issue: LC-1635

### CODE REVIEW

File: testing/test-mathjax.tsx [hunk 1]
```diff
@@ -23,11 +23,24 @@ type Props = {
...
     }, [tex, onRender]);
-    return <span dangerouslySetInnerHTML={{__html: domElement.outerHTML}} />;
+
+    React.useLayoutEffect(() => {
+        if (ref.current) {
+            addLabelWhenPresentational(ref.current);
+            ref.current.innerHTML = "";
+            ref.current.appendChild(domElement);
+        }
+    });
```
Comment (benchristel) [inline testing/test-mathjax.tsx:43]: Minor: Since `useLayoutEffect` callbacks run before `useEffect` callbacks, I wonder if this should be swapped with the `useEffect` block above, so the code executes in the same order that you read it.

**Review by benchristel — APPROVED** [review]
Looks good to me! I left a nonblocking comment inline - take it or leave it.

File: testing/test-mathjax.tsx (hunk 1 above)
Comment (nedredmond) [reply testing/test-mathjax.tsx:38]: swapped!
